src/FileLoaderHelper.h

```diff
@@ -19,7 +19,7 @@ using namespace dd4hep;
 
 // Function to download files
 inline void EnsureFileFromURLExists(std::string url, std::string file, std::string cache_str = "",
-                                    std::string cmd = "curl --retry 5 --location --fail {0} --output {1}")
+                                    std::string cmd = "curl --silent --retry 5 --location --fail {0} --output {1}")
```

Comment:
```suggestion
                                    std::string cmd = "curl --retry 5 --location --fail {0} --output {1}")
```

Reply:
Yeah, I agree. I wish there was a way to get rid of the progress bar which renders poorly and only get the final line.